Src/Private/Get-AbrCPPMService.ps1

```diff
@@ -0,0 +1,246 @@
+function Get-AbrCPPMService {
+    <#
+    .SYNOPSIS
+        Used by As Built Report to returns Service settings.
+    .DESCRIPTION
+        Documents the configuration of Aruba ClearPass in Word/HTML/Text formats using PScribo.
+    .NOTES
+        Version:        0.1.0
+        Author:         Alexis La Goutte
+        Twitter:        @alagoutte
+        Github:         alagoutte
+        Credits:        Iain Brighton (@iainbrighton) - PScribo module
+
+    .LINK
+        https://github.com/AsBuiltReport/AsBuiltReport.Aruba.ClearPass
+    #>
+    [CmdletBinding()]
+    param (
+
+    )
+
+    begin {
+        Write-PScriboMessage "Discovering Service settings information from $System."
+    }
+
+    process {
+        PageBreak
```

Comment:
Sections should immediately follow the previous section. Is there a particular reason a pagebreak is required?